docs.html

```diff
@@ -354,7 +354,355 @@ <h3>Floating Action Button</h3>
+                        </div>
+                    </div>
+                    <div class="code-snippet">
+                        <iframe class="code-snippet-iframe"
+                            src="https://carbon.now.sh/embed?bg=rgba%28151%2C58%2C153%2C1%29&t=seti&wt=none&l=htmlmixed&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522card-text-only%2522%253E%250A%2509%253Ch3%253ENike%253C%252Fh3%253E%250A%2509%253Ch2%253EJacket%253C%252Fh2%253E%250A%2509%253Cdiv%2520class%253D%2522card-rating%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522card-rating-stars%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fa%2520fa-star%2522%2520aria-hidden%253D%2522true%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fa%2520fa-star%2522%2520aria-hidden%253D%2522true%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fa%2520fa-star%2522%2520aria-hidden%253D%2522true%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fa%2520fa-star%2522%2520aria-hidden%253D%2522true%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fa%2520fa-star%2522%2520aria-hidden%253D%2522true%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522card-review%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253E257%2520reviews%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%2520%2520%2520%2520%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522card-price%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253ERs.%2520350%252F-%253C%252Fp%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522price-strike%2522%253ERs.%2520399%252F-%253C%252Fp%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522card-text%2522%253E%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cp%253E%2520There%2520is%2520no%2520better%2520brand%2520than%2520Nike%2520-%2520%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520known%2520for%2520its%2520functional%2520athleisure%2520and%2520sportswear%253C%252Fp%253E%2520%250A%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+                </article>
+                <!-- Card with badges and dismiss -->
+                <article class="box">
+                    <h3>Card with badges and dismiss</h3>
+                    <div class="box-content-row">
+                        <div class="card-text-container">
```

Comment:
These many divs really required?

Reply:
Yes. The basic structure is the article and inside this article I have a header field, a div to hold the component and another div to display the code snippet. So In the lines which you have pointed out the div with class name as "box-content-row" is the div which holds the component, I meant it is the placeholder to display the component.

Reply:
Okay, got it